### ISSUE TITLE
Italy outdated

### ISSUE THREAD

**Issue by yatil** [w3c/wai-policies-prototype#390]
According to this tweet, EN 301549 is “now extended to companies with an average balance of 900 million euros in the three-year period”. That means that EN 301549 is already applicable for some of Italy, and now even more. 

The Italy page does not mention any of this: https://www.w3.org/WAI/policies/italy/

Source: 

> Italian ICT #accessibility law (full EN 301549) is now extended to companies with an average balance of 900 million euros in the three-year period. If they do not adjust services after reporting, fine of 5% of balance. #a11y
> — [Roberto Scano](https://twitter.com/rscano/status/1284351364458582017)

https://twitter.com/rscano/status/1284351364458582017

Screenshot: 
![Screenshot of tweet quoted above](https://user-images.githubusercontent.com/58862/87921328-addd3e00-ca7a-11ea-9ab8-c44b6ee1da6c.png)

**Comment by maryjom** [comment]
@yatil Do you have any pointer to this law or decree? In searching the web I don't find anything.

**Comment by yatil** [comment]
No. I don’t know more than I put above. If I did I would have put it there :-D

**Comment by rscano** [comment]
Hi all! Reference is this decretee https://www.gazzettaufficiale.it/eli/id/2020/07/16/20G00096/sg that with article 29 update the "stanca act" (law 4/2004) extending for now to companies that have average balance more than 900 million euros in three-year period. For these companies, stanca act is applicable for procurement (need to buy ICT accessible produts), and web-mobile apps (contracts are null if they don't contain a clausole of respect of stanca act. Companies that will be noticed as inacessible to Italian Digital Agency will be recalled to fix in a given period. If they don't fix, they will be amended of a fee of maximum 5% of their balance.
One technical note: this is a "decreto legge" (law decretee) and this is active one day after  publication date (18 july) and need to be converted in law by parlament in 60 days. So this can have some changes. For example i will ask to apply also to a lower average balance level.
This was a proposal that i've done to the consultant of actual ministry for innovation and they finally agree to add this in last government act.

**Comment by iadawn** [comment]
@rscano Would you be able to submit the information regarding the Stanca Act in the [Policies Submission Form](https://www.w3.org/WAI/policies/submission/)? If you can add as much information in that form I can add it to the main entry.

**Comment by rscano** [comment]
Thank you i forward to Italian Digital Agency (AGID) so they can update.

 

---

Roberto Scano - Consulenze informatiche

Corte del Montello, 3 - 30132 Venezia (VE)

C.F.: SCNRRT75B05L736D - P.IVA 03378360279

Web site:  <http://robertoscano.info/> http://robertoscano.info 

skype:robertoscano

 

Da: Kevin White ***@***.***> 
Inviato: martedì 6 giugno 2023 12:57
A: w3c/wai-policies-prototype ***@***.***>
Cc: Roberto Scano ***@***.***>; Mention ***@***.***>
Oggetto: Re: [w3c/wai-policies-prototype] Italy outdated (#390)

 

@rscano <https://github.com/rscano>  Would you be able to submit the information regarding the Stanca Act in the Policies Submission Form <https://www.w3.org/WAI/policies/submission/> ? If you can add as much information in that form I can add it to the main entry.
